Add logging documentation for Elixir SDK (#9700)
getsentry/sentry-docs: docs/platforms/elixir/integrations/oban/index.mdx

  * Capturing errors and exceptions that happen in Oban jobs.
  * Cron monitoring for scheduled Oban jobs.

> This integration is built into the Sentry SDK since *v10.2.0*, and supports automatic monitoring since *v10.3.0*. It requires:

Reviewer: ```suggestion
This integration is built into the Sentry SDK starting with *v10.2.0*. It supports automatic monitoring starting with *v10.3.0*. It requires:
```

  1. Oban to be a dependency of your application.
  1. Oban version 2.17.6 or greater.

>   1. Elixir 1.13 or later, since that is required by Oban itself.

Author: Small additional info here.

> 
>   1. Oban to be a dependency of your application.
>   1. Oban version 2.17.6 or greater.
>   1. Elixir 1.13 or later, since that is required by Oban itself.

Reviewer: ```suggestion

  1. Oban version 2.17.6 and up to be a dependency of your application.
  1. Elixir version 1.13 and up, (since that's required by Oban).
```

    ]
  ]
```

> 
> This configuration will report failed Oban jobs to Sentry. It will also report started, completed, and failed job, alongside their duration. It will use the worker name as the `monitor_slug` of the reported cron.

Reviewer: ```suggestion

This configuration will report failed Oban jobs to Sentry. It will also report started, completed, and failed jobs, as well as their duration. It will use the worker name as the `monitor_slug` of the reported cron.
```
